packages/mobile/src/invite/saga.ts

```diff
@@ -327,10 +339,11 @@ export function* doRedeemInvite(inviteCode: string) {
       CURRENCY_ENUM.DOLLAR,
       SENTINEL_INVITE_COMMENT
     )
-    yield call(registerAccountDek, newAccount)
+    ValoraAnalytics.track(OnboardingEvents.invite_redeem_move_funds_complete)
+
     yield put(fetchDollarBalance())
     ValoraAnalytics.track(OnboardingEvents.invite_redeem_complete)
-    return true
+    return { success: true, newAccount }
```

Comment:
nit: What do you think of unifying the return type? for instance `{ success: false } | { success: true, newAccount: string }` or simply `string | null`?